test/end-to-end/core.sh

```diff
@@ -400,9 +400,10 @@ os::cmd::expect_success 'oc whoami'
 os::log::info "Testing deployment logs and failing pre and mid hooks ..."
 # test hook selectors
 os::cmd::expect_success "oc create -f ${OS_ROOT}/test/testdata/complete-dc-hooks.yaml"
-os::cmd::try_until_text 'oc get pods -l openshift.io/deployer-pod.type=hook-pre  -o jsonpath={.items[*].status.phase}' '^Succeeded$'
-os::cmd::try_until_text 'oc get pods -l openshift.io/deployer-pod.type=hook-mid  -o jsonpath={.items[*].status.phase}' '^Succeeded$'
-os::cmd::try_until_text 'oc get pods -l openshift.io/deployer-pod.type=hook-post -o jsonpath={.items[*].status.phase}' '^Succeeded$'
+os::cmd::expect_success 'oc rollout status dc/complete-dc-hooks'
+os::cmd::expect_success_and_text 'oc get rc/complete-dc-hooks-1 -o jsonpath={.metadata..annotations.openshift\\.io/deployment-hook-result\\.pre}' "success"
```

Comment:
`.metadata.annotations...` ?

Reply:
This worked for me:
```
oc get rc/complete-dc-hooks-1 -o jsonpath='{.metadata.annotations.openshift\.io/deployment-hook-result\.pre}'
```
yours didn't work

Reply:
@soltysh interesting... that test passed ;-)

Reply:
Depends which quotes you're using.

Reply:
@soltysh well single escape does not work in e2e test ;-) just got an failure because of that... so double quotes it is.